### DISCUSSION TITLE
OpenChakra Equivalent For Mantine

### DISCUSSION THREAD

**Discussion by anthonyalayo** [mantinedev/mantine#1398] [Feature requests]
Hey Everyone,

What do you think about something like this for Mantine?
https://openchakra.app/

Personally, this is what has me hooked on Chakra. I feel Mantine would take some serious market share with something like this.

**Comment by rtivital** [comment]
I doubt that dnd page editor is something that gives a library a huge advantage. It will always have limitations, and thus in 90% of the cases it cannot be used for anything serious.
